spec/plans/direct_spec.rb

```diff
@@ -23,6 +23,35 @@
       }
     end
 
+    let(:deployment_approved_response) do
+      {
+        'result' => {
+          'isPending' => false,
+          'approvalDecision' => 'APPROVED',
+        },
+        'error' => nil,
+      }
+    end
+
+    let(:deployment_declined_response) do
+      {
+        'result' => {
+          'isPending' => false,
+          'approvalDecision' => 'DECLINED',
+        },
+        'error' => nil,
+      }
+    end
+
+    let(:approval_pending_response) do
+      {
+        'result' => {
+          'isPending' => true,
+        },
+        'error' => nil,
+      }
+    end
+
```

Comment:
yey tests!

Reply:
Ideally we'd also have a test that verifies the deployment_pending_approval() function isn't called as part of the deployment. But I don't know how to do that